@3rdweb-sdk/react/hooks/usePayments.ts

```diff
@@ -862,3 +889,181 @@ export function usePaymentsSellerByAccountId(accountId: string) {
     { enabled: !!paymentsSellerId && !!address },
   );
 }
+
+export type PaymentsWebhooksType = {
+  id: string;
+  sellerId: string;
+  url: string;
+  isProduction: boolean;
+  createdAt: Date;
+};
+
+export function usePaymentsWebhooksByAccountId(accountId: string) {
+  invariant(accountId, "accountId is required");
+
+  const address = useAddress();
+  const { paymentsSellerId } = useApiAuthToken();
+  const [getWebhooksBySellerId] = useWebhooksBySellerIdLazyQuery();
+
+  const queryInfo = useQuery(
+    paymentsKeys.webhooks(accountId),
+    async () => {
+      invariant(paymentsSellerId, "no payments seller id found");
+      const { data, error } = await getWebhooksBySellerId({
+        variables: {
+          sellerId: paymentsSellerId,
+        } as WebhooksBySellerIdQueryVariables,
+      });
+
+      if (error) {
+        console.error(error);
+      }
+
+      if (data && data.webhook.length > 0) {
+        return data.webhook.map((webhook) => ({
+          id: webhook.id,
+          sellerId: webhook.seller_id,
+          url: webhook.url,
+          isProduction: webhook.is_production,
+          createdAt: new Date(webhook.created_at),
+        })) as PaymentsWebhooksType[];
+      } else {
+        return [] as PaymentsWebhooksType[];
+      }
+    },
+    { enabled: !!paymentsSellerId && !!address },
+  );
+
+  return {
+    ...queryInfo,
+    webhooks: queryInfo.data,
+  };
+}
+
+export const isValidWebhookUrl = (url: string) => {
+  if (!url || !url.startsWith("https://")) {
+    return false;
+  }
+
+  try {
+    const parsedUrl = new URL(url);
+    return !!parsedUrl.href && parsedUrl.protocol === "https:";
+  } catch (e) {
+    return false;
+  }
+};
+
+export type CreateWebhookInput = {
+  url: string;
+  isProduction: boolean;
+};
+
+export function usePaymentsCreateWebhook(accountId: string) {
+  invariant(accountId, "accountId is required");
+  const queryClient = useQueryClient();
+  const address = useAddress();
+  const { paymentsSellerId } = useApiAuthToken();
+
+  const [createWebhookBySellerId] = useInsertWebhookMutation({
+    refetchQueries: [WebhooksBySellerIdDocument],
+  });
+
+  return useMutationWithInvalidate(
+    async (input: CreateWebhookInput) => {
+      invariant(address, "No wallet address found");
+      invariant(paymentsSellerId, "No seller id found");
+      invariant(isValidWebhookUrl(input.url), "Invalid webhook url");
+
+      const webhookInput: Webhook_Insert_Input = {
```

Comment:
nit: if you put the object directly inside the return you get the type safety anyway and you import one less thing, but this also works